src/core/org/apache/jmeter/report/processor/graph/impl/LatencyOverTimeGraphConsumer.java

```diff
@@ -59,10 +94,12 @@ protected TimeStampKeysSelector createTimeStampKeysSelector() {
     @Override
     protected Map<String, GroupInfo> createGroupInfos() {
         HashMap<String, GroupInfo> groupInfos = new HashMap<>();
-        groupInfos.put(AbstractGraphConsumer.DEFAULT_GROUP, new GroupInfo(
-                new MeanAggregatorFactory(), new NameSeriesSelector(),
-                // We ignore Transaction Controller results
-                new LatencyValueSelector(false), false, false));
+        groupInfos.put("aggregate_report_max", //$NON-NLS-1$
```

Comment:
Is it regular to have max and pct ?

Reply:
It can be useful to identify the peak on outliers. Though it may not be relevant all the time it's better to have available and not need it than vice versa

Reply:
Easy enough to strip back to just pct though